doc/api/feature-flags.md

```diff
@@ -1,4 +1,4 @@
-<!-- Code generated by ./hack/doc/gen_feature_flags.sh - DO NOT EDIT -->
+<!-- Code generated by ./hack/doc/gen_feature_flags.sh - DO NOT EDIT -->\n
```

Comment:
Why the new line here?

Reply:
"\n" doesn't work on Fedora Linux. I will fix it.